src/domains/messages-common/99_locals.tf

```diff
@@ -17,4 +17,30 @@ locals {
 
   apim_v2_name             = "${local.product}-apim-v2-api"
   apim_resource_group_name = "${local.product}-rg-internal"
+
+  prefix    = "io"
+  env_short = "p"
```

Comment:
Prefix and env_short can be gotten from the same variables with var.prefix and var.env_short
```suggestion
```